ddtrace/profiling/collector/_lock.py

```diff
@@ -122,12 +122,8 @@ def acquire(self, *args, **kwargs):
             except Exception:
                 pass
 
-    def release(
-        self,
-        *args,  # type: typing.Any
-        **kwargs  # type: typing.Any
-    ):
-        # type: (...) -> None
+    def release(self, *args, **kwargs):
+        # type (typing.Any, typing.Any) -> None
```

Comment:
nit:
```suggestion
        # type: (typing.Any, typing.Any) -> None
```

Reply:
I tried applying this, but it causes the mypy linting to fail.

Reply:
@emmettbutler That's odd. Can we revert this to how it was prior to this PR then? i.e.: https://github.com/DataDog/dd-trace-py/blob/af0d087b2ba104845ddd3c7c794b5b1840fe9a44/ddtrace/profiling/collector/_lock.py#L125-L130